code/modules/research/xenobiology/crossbreeding/regenerative.dm

```diff
@@ -9,39 +14,118 @@ Regenerative extracts:
 	effect = "regenerative"
 	icon_state = "regenerative"
 
-/obj/item/slimecross/regenerative/proc/core_effect(mob/living/carbon/human/target, mob/user)
-	return
-/obj/item/slimecross/regenerative/proc/core_effect_before(mob/living/carbon/human/target, mob/user)
-	return
+	var/static/list/bodypart_replacement_table = list(
+		/obj/item/bodypart/arm/right = /obj/item/bodypart/arm/right/slime,
+		/obj/item/bodypart/arm/left = /obj/item/bodypart/arm/left/slime,
+		/obj/item/bodypart/leg/right = /obj/item/bodypart/leg/right/slime,
+		/obj/item/bodypart/leg/left = /obj/item/bodypart/leg/left/slime,
+		/obj/item/bodypart/head = /obj/item/bodypart/head/slime,
+		/obj/item/bodypart/chest = /obj/item/bodypart/chest/slime,
+	)
+
+/obj/item/slimecross/regenerative/proc/core_effect(mob/living/target, mob/user)
```

Comment:
Might as well make this `should_call_parent` if you go through all of them and add required parent calls

Reply:
But also not to knock you but you could've gotten around needing to add a parent call to every one of these by keeping the procs the way they were 

something like `afterattack` calls `heal_mob`, which calls `do_healing` and then `unique_core_effects` which handle the per color effects